cmd/experimental/posix/main.go

```diff
@@ -59,6 +59,7 @@ var (
 	origin                   = flag.String("origin", "", "Origin of the log, for checkpoints and the monitoring prefix.")
 	storageDir               = flag.String("storage_dir", "", "Path to root of log storage.")
 	pushbackMaxOutstanding   = flag.Uint("pushback_max_outstanding", tessera.DefaultPushbackMaxOutstanding, "Maximum number of number of in-flight add requests - i.e. the number of entries with sequence numbers assigned, but which are not yet integrated into the log.")
+	pushbackMaxDupesInFlight = flag.Uint("pushback_max_dupes_in_flight", 100, "Maximum number of number of in-flight duplicate add requests. When 0, duplicate entries are always pushed back.")
```

Comment:
I think we can remove `_in_flight` from the flag name. The `pushback_max_outstanding` flag is also for in-flight requests and we don't have "in flight" in the flag name.